k8s-manifests/minikube-any-namespace.yaml

```diff
@@ -201,6 +222,66 @@ spec:
               cpu: 0.05
               memory: 100Mi
 ---
+# Source: datacater/templates/pythonrunner.yaml
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: python-runner
+  labels:
+    app.kubernetes.io/part-of: datacater-0.1.0
+    app.kubernetes.io/name: "python-runner"
+    app.kubernetes.io/instance: release-name
+    app.kubernetes.io/version: "nightly-20230105"
+spec:
+  serviceName: pythonrunner
+  replicas: 5
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: "python-runner"
+      app.kubernetes.io/instance: release-name
+  template:
+    metadata:
+      name: python-runner
+      labels:
+        app.kubernetes.io/part-of: datacater-0.1.0
+        app.kubernetes.io/name: "python-runner"
+        app.kubernetes.io/instance: release-name
+        app.kubernetes.io/version: "nightly-20230105"
+    spec:
+      containers:
+        - name: python-runner
+          image: "datacater/python-runner:nightly-20230105"
+          imagePullPolicy: IfNotPresent
+          ports:
+            - name: http
+              containerPort: 50000
+          livenessProbe:
+            failureThreshold: 3
```

Comment:
We might want to adjust the failureTreshold to `10` as well here, similar to the readinessProbe.